### PR TITLE
Testing/add unit tests

### PR DESCRIPTION
**Pull request by dratenok**

### CODE REVIEW

File: src/test/java/com/cmlteam/util/UtilTest.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
+package com.cmlteam.util;
+
+import junit.framework.Assert;
+import junit.framework.TestCase;
```
Comment (xonixx) [inline src/test/java/com/cmlteam/util/UtilTest.java]: Please update Junit framework to latest version (5.x). It has API slightly changed, relies on annotations (like @Test) vs naming conventions.

File: src/test/java/com/cmlteam/util/UtilTest.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+import java.util.regex.Pattern;
+
+public class UtilTest extends TestCase {
+  public void testSafeToBoolean() {
+    Assert.assertTrue(Util.safeToBoolean(true));
+    Assert.assertTrue(Util.safeToBoolean("true"));
+    Assert.assertFalse(Util.safeToBoolean(false));
+    Assert.assertFalse(Util.safeToBoolean(null));
+    Assert.assertFalse(Util.safeToBoolean("TRUE"));
+    Assert.assertFalse(Util.safeToBoolean(new Object()));
```
Comment (xonixx) [inline src/test/java/com/cmlteam/util/UtilTest.java]: For conciseness please use static import for all Assert.assert* methods.

**Comment by xonixx** [comment]
On my machine 2 tests fail with similar error. Looks like the result depends on language / locale setting of operating system. Could we please adjust implementation to produce consistent results? 
image

File: src/test/java/com/cmlteam/util/UtilTest.java (hunk 1 above)
Comment (denis-ratenok) [reply src/test/java/com/cmlteam/util/UtilTest.java]: done

File: src/test/java/com/cmlteam/util/UtilTest.java (hunk 2 above)
Comment (denis-ratenok) [reply src/test/java/com/cmlteam/util/UtilTest.java]: done

**Comment by denis-ratenok** [comment]
> On my machine 2 tests fail with similar error. Looks like the result depends on language / locale setting of operating system. Could we please adjust implementation to produce consistent results?
> image

fixed

**Review by xonixx — APPROVED** [review]

**Pull request merged**